### PR TITLE
Feat/vist cpc 807 add start and endtime for visit

### PR DESCRIPTION
**Pull request by VladLoghin**
**JIRA:** link to jira ticket
## Context:
What is the ticket about and why are we doing this change.
## Does this PR change the .vscode folder in petclinic-frontend?:
If the PR changes the .vscode folder, explain why in detail because it should not. 
Be sure to include cgerard321 as a reviewer.

<span style="color:red">**Reviewers need to check for any changes to the 
.vscode folder and add a comment about it to their review comments.**</span>

## Changes
What are the various changes and what other modules do those changes affect.
This can be bullet point or sentence format.
## Before and After UI (Required for UI-impacting PRs)
If this is a change to the UI, include before and after screenshots to show the differences.
If this is a new UI feature, include screenshots to show reviewers what it looks like. 
## Dev notes (Optional)
Specific technical changes that should be noted
## Linked pull requests (Optional)
Pull request links

### CODE REVIEW

**Comment by VladLoghin** [comment]
Added a vistStartDate and a visitEndDate

File: api-gateway/src/main/java/com/petclinic/bffapigateway/presentationlayer/v2/VisitController.java [hunk 1]
```diff
@@ -38,14 +38,6 @@ public ResponseEntity<Flux<VisitResponseDTO>> getAllVisits() {
         return ResponseEntity.ok().body(visitsServiceClient.getAllVisits());
     }
 
-    @SecuredEndpoint(allowedRoles = {Roles.ADMIN})
-    @PostMapping(value = "", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
-    public Mono<ResponseEntity<VisitResponseDTO>> addVisit(@RequestBody Mono<VisitRequestDTO> visitResponseDTO) {
-        return visitsServiceClient.addVisit(visitResponseDTO)
```
Comment (Zako563) [inline api-gateway/src/main/java/com/petclinic/bffapigateway/presentationlayer/v2/VisitController.java:44]: what happened to this method, why its deleted?

**Comment by Zako563** [comment]
You did not fill up the pull request "form" :
image


You can find all the information you need in your Jira ticket.

File: cart-service/lombok.config [hunk 2]
```diff
@@ -0,0 +1 @@
+lombok.addLombokGeneratedAnnotation = true
```
Comment (AlejandroBrnab) [inline cart-service/lombok.config:1]: why did you added this in the cart service? It didn't have it before?

**Comment by ioannisp03** [comment]
Naming convention for the PR needs to be : feat/(VIST-CPC-807): Add Start And Endtime For Visit

**Comment by Rrregine** [comment]
It seems you have touched .vscode folder: 
{01B1C8E8-023A-415F-99BA-3DC974456A5B}
Maybe it is better to add Christine as reviewer, just to make sure everything is correct.

In the visits-service, I see that you mostly renaming fields (Date -> StartDate). However, a lot of other services and configuration files are involved in your PR, but they seem to be the same as before changing? Do you know what happened to these files?

**Pull request closed without merging**